jvm-libs/linea/besu-libs/build.gradle

```diff
@@ -3,27 +3,30 @@ plugins {
   id 'java-library'
 }
 
-def besuArtifactGroup="org.hyperledger.besu"
+//def besuArtifactGroup="org.hyperledger.besu"
+//def besuVersion=libs.versions.besu.get()
+def besuArtifactGroup="io.consensys.linea-besu"
+def besuVersion="24.12-develop-2098dd2"
```

Comment:
Would it be harmful if we set this version in `libs.versions.toml` temporarily instead?

Reply:
that was my first try, it broke the build because this version is not availe in default hyperledger besu group. It would require more build refactors and effort without knowing it works. So leaving as is for now.